src/terra-dev-site/tool/terra-dev-site/WebpackLoaders.e/ExampleLoader.b.tool.mdx

```diff
@@ -2,8 +2,9 @@ import Example from '../../../test/loaders/example?dev-site-example';
 
 # Example Loader
 
-The Example loader will take the passed in file, add it to an example template then location the source for the file in the src folder and render the example output.
-The file must be a return a react component that has no required props to render.
+The Example loader will take the passed in file, add it to an example template then locate the source for the file in the src folder and render the example output.
+The file must be a return a react component that has no required props to render. The Example loader will also look for surface level css files and search for
+the file in the same directory as the source file in order to render it.
```

Comment:
The file should not have to be in the same directory

Reply:
Updated description [here](https://github.com/cerner/terra-dev-site/pull/281/commits/f4ff529e1f5620f5daf36cf5aaf4f9acab585291), currently the file finder can find paths starting with `./` and explicit paths, relative paths (`../../../`) are currently a wip.